src/components/components/teaser-box/teaser-box.js

```diff
@@ -30,7 +30,12 @@ export const TeaserBox = props => {
 
   return (
     <div className={classes.join(" ")}>
-      <a href={target} className="teaser-box__wrapper-link">
```

Comment:
I don't remember if this is an external link or not, could you verify that this is an external link?

Reply:
yes sure

Reply:
This is not an external link so i removed that from here